compiler/src/iree/compiler/GlobalOptimization/SetEncoding.cpp

```diff
@@ -128,6 +128,16 @@ static MatmulNarrowSizes getMatmulNarrowSizes(ShapedType outType,
   if (!ShapedType::isDynamic(N) && N < kNarrowThreshold) {
     narrow.N = N;
   }
+
+  // only pick 1 if both are present
+  if (narrow.M && narrow.N) {
+    // if M is the larger one, pick M
```

Comment:
The code is correct but the comment is the other way around!
Pick the smaller of M or N; in case of a tie, prefer M to avoid later transposition of narrow N into narrow M.

Reply:
Clearly I do not understand English.